src/CppUTestExt/MockFailure.cpp

```diff
@@ -104,10 +104,15 @@ MockExpectedCallsDidntHappenFailure::MockExpectedCallsDidntHappenFailure(UtestSh
 MockUnexpectedCallHappenedFailure::MockUnexpectedCallHappenedFailure(UtestShell* test, const SimpleString& name, const MockExpectedCallsList& expectations) : MockFailure(test)
 {
     int amountOfExpectations = expectations.amountOfExpectationsFor(name);
-    if (amountOfExpectations)
```

Comment:
hi, would it be better if replacing temp variables `amountOfExpectations` and `actualCallCount`with directly function call ``expectations.amountOfExpectationsFor``?  The direct function is already clear and those temp variables is not indispensable.